bazel/osx/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"] + ["**/*"]),
+)
+
+exports_files([
+    "xcode_locator.m",
+])
+
+DARWIN_XCODE_LOCATOR_COMPILE_COMMAND = """
+  /usr/bin/xcrun clang -fobjc-arc -framework CoreServices \
+      -framework Foundation -o $@ $<
+"""
+
+genrule(
```

Comment:
This combined with the code in #1348 should let you build and test on OSX. By itself it just shouldn't break the linux build, I guess.

I'll try to remove some stuff from bazel/osx again -- I had trouble before that I though was related to sandboxing, but I may have been mistaken.

Will add a note about installing realpath.